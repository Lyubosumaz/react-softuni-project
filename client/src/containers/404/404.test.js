import React from 'react';
import FourOFour from './404';
import { shallow } from 'enzyme';

it('Renter 404 component', () => {
    const wrapper = shallow(<FourOFour />);
    expect(wrapper).toBeTruthy();
});

it('Checks for html changes', () => {
    const wrapper = shallow(<FourOFour />);
    expect(wrapper).toMatchSnapshot();
});
