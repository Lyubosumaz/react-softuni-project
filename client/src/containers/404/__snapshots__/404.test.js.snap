// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checks for html changes 1`] = `
<div
  className="error-container"
>
  <h1>
    Oops! Page not found
  </h1>
  <h2>
    404
  </h2>
  <p>
    <b>
      We can't find the page you're looking for
    </b>
  </p>
  <button
    className="active-button"
    name="404"
    onClick={[Function]}
  >
    Go Back Home
  </button>
</div>
`;
